tests/lax_numpy_test.py

```diff
@@ -2119,6 +2119,28 @@ def testCov(self, shape, dtype, rowvar, ddof, bias, rng_factory):
     self._CompileAndCheck(lnp_fun, args_maker, check_dtypes=True, atol=tol,
                           rtol=tol)
 
+  @parameterized.named_parameters(
+      jtu.cases_from_list(
+        {"testcase_name": "_shape={}_shape2={}_dtype={}_mode={}".format(
+            shape, shape2, dtype, mode),
+         "shape": shape, "dtype": dtype, "mode": mode,
+         "shape2": shape2,"rng_factory": rng_factory}
+        for shape in [(i,) for i in range(2, 100)]
+        for shape2 in [(i,) for i in range(2, 100)]
+        for dtype in [onp.float16, onp.float32, onp.float64]
```

Comment:
I'd use `float_dtypes` here rather than enumerating a list of types. Not all hardware platforms support all of these types (e.g., TPU doesn't support `float16`.)